command/format.go

```diff
@@ -58,6 +67,14 @@ var Formatters = map[string]Formatter{
 	"yml":   YamlFormatter{},
 }
 
+func Format() string {
+	format := os.Getenv(EnvVaultFormat)
```

Comment:
Could we retrieve this as part of an init function instead of doing the envvar lookup each time?

Reply:
We just got away from init functions in favor of being able to intercept the args earlier in the main run function. This isn't going to be meaningfully slow for interactive uses, is there a reason not to just call this a couple of times?

Reply:
I've experienced `Getenv` being slow (in relative terms), particularly on Windows systems. I wonder if we could use a Once instead and cache the result?